discordBot.js

```diff
@@ -43,6 +43,47 @@ class DiscordBot {
   stop() {
     return this.client.destroy();
   }
+
+  // send files to discord channel given the file path and channel
+  sendFile(filePath, channelId){
+    this.client.on('ready', () => {
+      var attachment = new MessageAttachment(filePath);
+      this.client.channels.cache.get(channelId).send(attachment);
+      return 'success';  
+    })    
+  }
+
+  //sends a link to a google form to the specifies channel
+  sendGoogleForm(googleFormUrl, channelId){
+    this.client.on('ready', () => {
+      this.client.channels.cache.get(channelId).send(googleFormUrl);
+      return 'success';
```

Comment:
same as above

Reply:
same for as sendfile need the on ready otherwise channels.cache.get..........send(googleformurl) is undefined since cache would be empty